rubin_scheduler/skybrightness_pre/sky_model_pre.py

```diff
@@ -73,6 +76,34 @@ def interp_angle(x_out, xp, anglep, degrees=False):
     return result
 
 
+def simple_daytime(sky_alt, sky_az, sun_alt, sun_az, filter_name="r", bright_val=2.0, sky_alt_min=20.0):
+    """A simple function to return a sky brightness map when the sun is up
+
+    Parameters
+    ----------
+    sky_alt : `float`
+        Altitude of poistion(s) on the sky. Degrees.
+    sky_az : `float`
+        Azimuth of poistion(s) on the sky. Degrees.
+    sun_alt : `float`
+        Altitude of the sun. Degrees.
+    sun_az : `float`
+        Azimuth of the sun. Degrees.
+    filter_name : `str`
+        Name of the filter, default "r". Currently unused, but
+        could be useful in the future if a more complicated function
+        gets subbed in.
+    bright_val : `float`
+        The value to plug into the sky. Default 2 mag/sq arcsec.
+    sky_alt_min : `float`
+        Set all sky below the alt limit to NaN. Default 20 degrees.
+    """
+
+    result = sky_alt * 0 + np.nan
```

Comment:
`np.full_like(sky_alt, np.nan)` might make it even more obvious what you're up to.